### PR TITLE
RAD-90 Refactor RadiologyObsFormController Including Tests

### PR DESCRIPTION
**Pull request by bgeVam**
see https://issues.openmrs.org/browse/RAD-90

### CODE REVIEW

File: api/src/main/java/org/openmrs/propertyeditor/ObsEditor.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	 * @see java.beans.PropertyEditorSupport#setAsText(java.lang.String)
+	 */
+	public void setAsText(String text) throws IllegalArgumentException {
+		ObsService obsService = Context.getObsService();
+		if (StringUtils.hasText(text)) {
+			try {
+				setValue(obsService.getObs(Integer.valueOf(text)));
+			}
+			catch (Exception exception) {
+				Obs Obs = obsService.getObsByUuid(text);
```
Comment (sunbiz) [inline api/src/main/java/org/openmrs/propertyeditor/ObsEditor.java]: Why not do this more clearly getObs(int) or getObsByUuid(text) using a ternary operator?

Comment (bgeVam) [inline api/src/main/java/org/openmrs/propertyeditor/ObsEditor.java]: My intention was to stick with the way the editors are implemented in the core:
see 
https://github.com/openmrs/openmrs-core/blob/1.9.x/api/src/main/java/org/openmrs/propertyeditor/OrderEditor.java

Comment (bgeVam) [inline api/src/main/java/org/openmrs/propertyeditor/ObsEditor.java]: could you be more precise what you want to explain?

Comment (sunbiz) [inline api/src/main/java/org/openmrs/propertyeditor/ObsEditor.java]: Nevermind, I think its fine, if thats the convention with other xxxEditor

**Pull request merged**